### ISSUE TITLE
git rebase diagram correct?

### ISSUE THREAD

**Issue by Bascy** [ByteByteGoHq/system-design-101#34]
I think the `git rebase` diagram is not correct:

After rebasing the feature branch on master, the master branch does not change (still pointing to D), feature branch is still pointing to `G` but commit `E` is now based on `D` in stead of `B`

**Comment by macie** [comment]
@Bascy you are right. Rebase changes "base commit" of feature branch, without merging feature branch to main branch.
